components/brave_sync/ui/components/modals/viewSyncCode.tsx

```diff
@@ -87,12 +86,12 @@ export default class ViewSyncCodeModal extends React.PureComponent<Props, State>
   }
 
   onConfirmDismissModal = () => {
-    const { devices, isSyncConfigured } = this.props.syncData
+    const { isSyncConfigured } = this.props.syncData
     // sync is enabled when at least 2 devices are in the chain.
     // this modal works both with sync enabled and disabled states.
     // in case user opens it in the enabled content screen,
     // check there are 2 devices in chain before reset
```

Comment:
The comment needs an update

Reply:
This will be done in context of separate issue  https://github.com/brave/brave-browser/issues/6941 .